### DISCUSSION TITLE
Delete/Create Collection

### DISCUSSION THREAD

**Discussion by jordanTMcfarland** [Gremlinq/ExRam.Gremlinq#146] [Q&A]
Does Gremlinq expose Gremlin.Net's ability to delete and create collections?

**Comment by danielcweber** [comment]
I'm not sure what Gremlin.Net's ability to delete and create collections is.

**Reply by jordanTMcfarland** [reply]
Hi Daniel - I realized I misunderstood an example I was reviewing; thus, making my question a mute point. Apologies

**Reply by jordanTMcfarland** [reply]
Following up on this... I found what inspired my original question. It is actual from the CosmosDatabase class in Azure.Cosmos package. It looks like a nice option to verify a container is within DB prior to interacting with it.

```
public abstract Task<ContainerResponse> CreateContainerIfNotExistsAsync(
      string id,
      string partitionKeyPath,
      int? throughput = null,
      RequestOptions requestOptions = null,
      CancellationToken cancellationToken = default (CancellationToken));
```
